app.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const app = express();
+
+app.use(express.static('public'));
+
+app.get('/home', (request, response, next) => response.sendFile(__dirname + '/views/home-page.html'));
```

Comment:
Hey Ash, I sugest you to just use the "/" for your home page, like: `app.get('/home', (request, response, next) => response.sendFile(__dirname + '/views/home-page.html'));`

Like this as soon you start the server you'll see the home page.